src/main/webapp/WEB-INF/pages/chats/list.jsp

```diff
@@ -0,0 +1,60 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+
+<t:template>
+    <script language="javascript" type="text/javascript">
+        tr.object = "<spring:message code="js.chat"/>"
+    </script>
+
+    <ol class="breadcrumb">
+        <li><a href="/subjects"><spring:message code="subject.subject"/>: ${subjectName}</a></li>
+        <li class="active"><a href="#"><spring:message code="title.chats"/></a></li>
+    </ol>
+      <H1><spring:message code="title.chats"/></H1>
+
+      <sec:authorize access="isAuthenticated()">
+        <p><a href="chats/add" class="btn btn-success"><spring:message code="chat.add_chat"/></a></p>
+      </sec:authorize>
+
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
looks like a nice place to have a practice with tags ;) (just like you did with the `<t:template/>` tag